app/src/androidTest/java/org/mozilla/focus/screengrab/DownloadScreenshot.java

```diff
@@ -209,9 +212,11 @@ public void screenshotDownload() {
                 .inRoot(RootMatchers.isPlatformPopup())
                 .perform(click());
 
+        SystemClock.sleep(MockUIUtils.POPUP_DELAY);
```

Comment:
Just wanted to note: SystemClock.sleep should only be used in Screengrab, not in regular UI test. Or we'll have too many flaky tests.